src/Error/ExceptionRenderer.php

```diff
@@ -184,7 +184,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```

Comment:
The error logs are always written to error.log. However, Unauthorized Exceptions are a 'HttpException' which is not logged by default. You can change which exceptions are logged using the `Exception.skipLog` & `Exception.log` config options.